CONTRIBUTING.md

```diff
@@ -0,0 +1,83 @@
+# CONTRIBUTING.md
+
+## How to start?
+
+Great, so you want to join the development!
+
+First, [set up a development environment](INSTALL.md). Since you're going to
+write new code, use the `docker-compose.dev.yml` method.
+
+If everything went right, the system should be accessible at `localhost:80`.
+
+## Development workflow
+
+We use a standard [github fork workflow](
+https://gist.github.com/Chaser324/ce0505fbed06b947d962).
+
+1. Fork the repository.
+
+2. Create a new branch. The name does not matter, but the recommended format
+  is `feature/xxx` or `fix/yyy`.
+
+3. Work on your changes!
```

Comment:
Yes, that is guaranteed by the developer docker-compose (assuming it works correctly :sweat_smile:) Shouting because I'm enthusiastic for writing code! :P